### ISSUE TITLE
zappa stops installing precompiled packages if an exception is raised processsing any package

### ISSUE THREAD

**Issue by ozgurakcali** [Miserlou/Zappa#2057]
## Context
When a cached wheel file for a precompiled package is corrupted, zappa stops processing precompiled packages altogether. 

## Expected Behavior
Zappa may skip the current package, but should continue with other packages in the list

## Actual Behavior
Zappa stops processing precompiled packages

## Possible Fix
Exception handling [here](https://github.com/Miserlou/Zappa/blob/988ba099c22c66b79257cf9b99ec100d264b5b1e/zappa/core.py#L651) could be moved into the for loop, so that if a package fails, process continues with other packages

## Steps to Reproduce
1. Place a corrupted wheel file for an installed package in the cache directory (like psycopg2)
2. run zappa update command

## Your Environment
* Zappa version: 0.50.0
* Operating System and Python version: Mac OS, Python 3.7

**Comment by jneves** [comment]
I'm trying to think of a situation where I'd want the packaging to continue. If there's a broken package it might be because of a failed download or user error, but in the end we'd end up with a broken and more difficult to debug packaging failure, right?

**Comment by ozgurakcali** [comment]
@jneves yes, but what happens in this case is, packaging continue, just that it stops using the rest of precompiled packages. Maybe packaging could stop altogether, displaying an error message related to the Python package that caused the problem.
